Userland/Libraries/LibGfx/PaintStyle.h

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright (c) 2023, MacDue <macdue@dueutil.tech>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/Function.h>
+#include <AK/NonnullRefPtr.h>
+#include <AK/QuickSort.h>
+#include <AK/RefCounted.h>
+#include <AK/RefPtr.h>
+#include <AK/Vector.h>
+#include <LibGfx/Color.h>
+#include <LibGfx/Forward.h>
+#include <LibGfx/Gradients.h>
+#include <LibGfx/Rect.h>
+
+namespace Gfx {
+
+struct PaintStyle : RefCounted<PaintStyle> {
```

Comment:
We typically only use `struct` for simple structures with a handful of public members.